.github/workflows/release_portable_linux_pytorch_wheels.yml

```diff
@@ -24,8 +26,21 @@ on:
   workflow_dispatch:
     inputs:
       amdgpu_family:
-        required: true
+        type: choice
+        options:
+          - gfx110X-dgpu
+          - gfx1151
+          - gfx120X-all
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
+      test_runs_on:
+        description: |
+          The runner label to use for tests. Leave this empty to skip testing.
+          See available options for each associated family at
+          https://github.com/ROCm/TheRock/blob/main/build_tools/github_actions/amdgpu_family_matrix.py
         type: string
+        default: linux-mi325-1gpu-ossci-rocm
```

Comment:
I am a bit concerned that having a default will result in building for one family from the choices but missing to set the correct label here and testing on the wrong family afterwards. Not having a default would mean to know / look up one more label. Though I am not blocking on this even though still think it would be nice to not know the labels at all.

If testing can be skipped here, how does this colludes with #1072?

Reply:
Okay, I can try restoring the script we had and fixing it. We're still not running pytorch tests on Windows until this lands though, since gfx1151 is not triggering test jobs :/

Reply:
Thanks!